### PR TITLE
Updated readme for adding words to wordlist

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
-packaged version of the dictionaries from
-https://build.opensuse.org/package/show/Documentation:Tools/suse-documentation-dicts-en.
+* For openSUSE Leap and Tumbleweed users, the easiest option is installing the
+  packaged version of the dictionaries from
+  https://build.opensuse.org/package/show/Documentation:Tools/suse-documentation-dicts-en.
+* For Debian and Ubuntu users, find the package at
+  https://build.opensuse.org/package/show/home:tbazant/suse-documentation-dicts
 
 
 ### Building the Dictionaries
```
Comment (dariavladykina) [inline README.md:20]:
```suggestion
### Building the dictionaries
```

File: README.md [hunk 2]
```diff
@@ -42,13 +44,13 @@ make install
 ```
 
 
-### Using the Dictionaries
+## Using the Dictionaries
```
Comment (dariavladykina) [inline README.md:47]:
```suggestion
## Using the dictionaries
```

File: README.md [hunk 3]
```diff
@@ -84,21 +86,21 @@ Add the following lines to your Emacs configuration file:
...
 
-#### Hunspell
+### Hunspell
 
 ```
 hunspell -H -i utf-8 -d en_US,en_US-suse-doc FILE
 ```
 
-#### Vim Word List & Vim
+### Vim Word List & Vim
```
Comment (dariavladykina) [inline README.md:95]:
```suggestion
### Vim word list & Vim
```

File: README.md [hunk 4]
```diff
@@ -141,34 +143,67 @@ is available from (http://www.xmlmind.com/xmleditor/dictbuilder.shtml).
 
 ### Adding Words
```
Comment (dariavladykina) [inline README.md:144]:
```suggestion
### Adding words
```

File: README.md [hunk 5]
```diff
@@ -141,34 +143,67 @@ is available from (http://www.xmlmind.com/xmleditor/dictbuilder.shtml).
...
-The word list format supports suffixes dictionary words to avoid word
-repetitions. Suffixes are appended to the regular entry this way:
-
-```
-entry +suffix +y/iesuffix
-```
+The doctionaries include a word list of correct words that are not part of the
+standard English dictionaries. The word list is being extended as the writers
+find new words that are not yet part of the word list but still are correct. If
+you want to contribute to improving the spell-checking experience, collect such words that the spell checker marks as incorrect but you consider them correct.
```
Comment (dariavladykina) [inline README.md:149]:
```suggestion
The dictionaries include a word list of correct words that are not part of the
standard English dictionaries. The list of acceptable words is updated as 
writers discover words that are correct but not yet included. To improve 
spell checking, please collect words that the spell checker flags as incorrect 
but you believe are correct.
```

File: README.md [hunk 6]
```diff
@@ -141,34 +143,67 @@ is available from (http://www.xmlmind.com/xmleditor/dictbuilder.shtml).
...
+
+TIP: If you are using VSCode, you probably have such file already. Check if the
+`SUSE-adWords-dict.dic` file exists. If yes, VSCode lets you add the suspect
+word to this dictionary by clicking the corresponding small yellow bulb next to
+the affected line.
```
Comment (dariavladykina) [inline README.md:154]:
```suggestion
TIP: If you are using VSCode, you probably have such a file already. Check if the
`SUSE-adWords-dict.dic` file exists. If yes, VSCode lets you add the suspect
word to this dictionary by clicking the corresponding small yellow bulb next to
the affected line.
```

File: README.md [hunk 7]
```diff
@@ -141,34 +143,67 @@ is available from (http://www.xmlmind.com/xmleditor/dictbuilder.shtml).
...
+
+Once you collect a certain amount of such words, follow these steps:
```
Comment (dariavladykina) [inline README.md:156]:
```suggestion
Once you collect a certain amount of new words, follow these steps:
```

File: README.md [hunk 8]
```diff
@@ -141,34 +143,67 @@ is available from (http://www.xmlmind.com/xmleditor/dictbuilder.shtml).
...
+    This produces the following three words in the output word list:
+
+    ```
+    entry
+    entrysuffix
+    entriesuffix
+    ```
+
+    * There must be exactly one space before each suffix definition.
+    * Suffix definition start with a `+` character.
```
Comment (dariavladykina) [inline README.md:191]:
```suggestion
    * The suffix definition starts with a `+` character.
```

File: README.md [hunk 9]
```diff
@@ -141,34 +143,67 @@ is available from (http://www.xmlmind.com/xmleditor/dictbuilder.shtml).
...
+    ```
+    make sortvalid
+    ```
 
-```
-entry
-entrysuffix
-entriesuffix
-```
+  6. Commit the changes to your branch.
```
Comment (dariavladykina) [inline README.md:203]:
```suggestion
  6. Commit the changes to your branch:
```

File: README.md [hunk 10]
```diff
@@ -1,27 +1,29 @@
-## Spellcheck Dictionary for SUSE Documentation
+## Spellcheck dictionaries for SUSE documentation
```
Comment (dariavladykina) [inline README.md:1]:
```suggestion
## Spell-check dictionaries for SUSE documentation
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see suggestions for this README. Thanks!

**Review by tomschr — APPROVED** [review]
I briefly glanced over the changes and it looks good.
Thanks Daria & Tomas! :+1:

**Pull request merged**